src/basho_bench_measurement.erl

```diff
@@ -46,9 +45,6 @@
 start_link() ->
     gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
 
-run() ->
-    gen_server:cast(?MODULE, run).
-
```

Comment:
I'm not sure it makes sense to remove the `run` logic from this module and `basho_bench_stats` and whatever else. This provides a way to separate bootstrapping from actually running the benchmarks. Removing potentially opens up the possibility of starting to gather stats before the benchmark is actually running.

I think we should really have two top level functions, `setup_benchmark` and `run_benchmark`, so that we could isolate the two and provide control to whatever application is wrapping this. Doing that would require we retain the `run` triggers.